packages/ios-client/Sources/FishjamClient/FishjamClientInternal.swift

```diff
@@ -60,22 +579,26 @@ internal class FishjamClientInternal: MembraneRTCDelegate, WebSocketDelegate {
             break
         case .pong(_):
             break
+        //TODO: worth consider should reconnect happend here
```

Comment:
Do we have any ticket for this? I would either add ticket number here or remove this todo

Reply:
TODO is not TODOING anymore